exasol/nb_connector/cli/processing/processing.py

```diff
@@ -58,3 +61,20 @@ def save(
         if secret := option.get_secret(interactive=bool(value)):
             scs.save(option.scs_key, secret)
     return 0
+
+
+def show_scs_content(scs_file: Path) -> int:
```

Comment:
Why not throwing an exception here, but returning an int. Looks like ancient C code ;)  ....

Reply:
Currently, the [CLI layer](https://github.com/exasol/notebook-connector/blob/feature/269-Implemented_showing_saved_options/exasol/nb_connector/cli/commands/show.py#L17-L18) can use 
```python
result = processing.show_scs_content(scs_file)
sys.exit(result)
```

With an exception, the code would get longer and require more tests on the integration level: 
```python
try:
    processing.show_scs_content(scs_file)
except Exception:
    print(...)
    sys.exit(1)
```